DataFormats/CTPPSDigi/src/classes_def.xml

```diff
@@ -14,7 +14,8 @@
   </class>
   <class name="edm::Wrapper<TotemTriggerCounters>"/>
 
-  <class name="TotemVFATStatus" ClassVersion="3">
+  <class name="TotemVFATStatus" ClassVersion="4">
+   <version ClassVersion="4" checksum="1562789855"/>
     <version ClassVersion="2" checksum="3003040825"/>
     <version ClassVersion="3" checksum="1439138260"/>
   </class>
```

Comment:
the following are present in 9X version but not here
```
<class name="edm::DetSet<TotemVFATStatus>"/>
<class name="std::vector<TotemVFATStatus>"/>
<class name="std::vector<edm::DetSet<TotemVFATStatus> >"/>
```
was it dropped in backport for a reason?